### ISSUE TITLE
Clarify which fields are optional and which fields are not

### ISSUE THREAD

**Issue by poscat0x04** [standardnotes/docs#48]
Hi, I'm trying to implement a standard notes sync server and would like to know which fields are optional and which fields are not. The documentation only said that the `api` field "must not be null" but there are no such comments for other fields. I doubt that they are all required cause surely a deleted item does not need to have a `content` field. It would be nice if the documentation could clear things up.

**Comment by JaspalSuri** [comment]
Hi @poscat0x04, thank you for the suggestion. Are you referring to [this page](https://docs.standardnotes.com/self-hosting/configuration-options) in the documentation? 

[[Internal task link for the dev team.]](https://app.asana.com/0/1160981203939075/1201256907596671/f)

**Comment by poscat0x04** [comment]
No, I'm talking about https://docs.standardnotes.com/specification/auth and https://docs.standardnotes.com/specification/sync

**Comment by JaspalSuri** [comment]
Thank you for clarifying! I've updated our internal task's description accordingly. 🙂

**Comment by karolsojko** [comment]
Hi @poscat0x04 as mentioned in this comment: https://github.com/standardnotes/docs/issues/1#issuecomment-589978343 the documentation in the links you've mentioned is outdated. We plan to refactor this in the near future, unfortunately as mentioned in the latest email we have our hands full with some exciting new changes coming in :) I'll keep this issue open so that we can address it in future.
